scratch/PCA/extract_data.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# In[61]:
+
+#!/usr/bin/env python
```

Comment:
Yes. Since our last exchange, we should prefer actions/pcr.py


--
Henry




> On Aug 29, 2016, at 3:00 PM, Marc Daya <notifications@github.com> wrote:
> 
> In scratch/PCA/extract_data.py <https://github.com/csm-adapt/tracr/pull/8#discussion_r76686429>:
> 
> > @@ -0,0 +1,123 @@
> > +
> > +# coding: utf-8
> > +
> > +# In[61]:
> > +
> > +#!/usr/bin/env python
> This script looks very similar to tracr/actions/pcr.py; is one of these to be preferred over the other?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/csm-adapt/tracr/pull/8/files/0e789d0b796025a56d8f9ff7715edf7236627273#r76686429>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AMZvXfSnId9HG_OOpypT0dnZJNO7iI7Jks5qk0iBgaJpZM4JvDIc>.
>